### PR TITLE
[Bugfix]Svgr component type not recognized

### PR DESCRIPTION
**Pull request by LJKaski**
SvgrComponent type was required, but not available in the build. Changed from awesome typescript loader (repository archived in github, so no longer developed) to ts-loader. Also changed mode from development to production in webpack config.

Changed the way of implementing the custom style so that it gets included in the build. It was hard to find a squeaky clean solution, but since we are talking about an internal typing issue, this solution should do just fine.

Tested with a simple SSR template. If you have the chance, give it a go with an SSR template as well.

### CODE REVIEW

**Comment by ketsappi** [comment]
Didn't seem to be working as wanted.


## Steps how to reproduce & error
1.) Tested by creating a new project with create-react-app and Typescript option
`npx create-react-app tst --typescript`

2.) Added the suomifi-icons of this branch as dependency

3.) Added "bus" icon
`<SuomifiIcon color="orange" icon="bus" width={80} height={80} />`

4.) From the `tsconfig.json` changed `skipibLibCheck` to **false**

### Error
```
Failed to compile.

/tst/node_modules/suomifi-icons/dist/icons.d.ts
TypeScript error in /tst/node_modules/suomifi-icons/dist/icons.d.ts(1,22):
File '/tst/node_modules/suomifi-icons/typings/index.d.ts' not found.  TS6053

  > 1 | /// <reference path="../typings/index.d.ts" />
      |                      ^
    2 | export declare type IconKeys = keyof typeof icons;
    3 | export declare const icons: {
    4 |     arrowUp: import("*.svg").SvgrComponent;
```

But if not changing the `skibLibCheck`, it will run just fine.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "suomifi-icons",
-  "version": "0.0.12",
+  "version": "0.0.13",
```
Comment (aappoalander) [inline package.json:3]: Version changes should be done using separate commit and PR.

File: tsconfig.json [hunk 2]
```diff
@@ -46,7 +46,8 @@
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     "typeRoots": [
       "node_modules/@types",
-      "types"
+      "types",
+      "typings"
     ] /* List of folders to include type definitions from. */,
     // "types": [],                           /* Type declaration files to be included in compilation. */
```
Comment (aappoalander) [inline tsconfig.json:52]: Should this rather be completely removed?

File: tsconfig.json [hunk 3]
```diff
@@ -46,7 +46,8 @@
     // "rootDirs": [],                        /* List of root folders whose combined content represents the structure of the project at runtime. */
     "typeRoots": [
       "node_modules/@types",
-      "types"
+      "types",
+      "typings"
```
Comment (aappoalander) [inline tsconfig.json:50]: What is the difference here? Why 2 separate local folders?

File: package.json (hunk 1 above)
Comment (LJKaski) [reply package.json:3]: Ah, forgot to change that back after testing with verdaccio.

File: tsconfig.json (hunk 2 above)
Comment (LJKaski) [reply tsconfig.json:52]: Yes, I think it should. The types folder was there by default and didn't check for redundancy when adding the new one.

File: tsconfig.json (hunk 3 above)
Comment (LJKaski) [reply tsconfig.json:50]: See the comment above.

Comment (aappoalander) [reply tsconfig.json:50]: Sure, my proposal would be to get rid of our custom one if it works using only e.g. types.

Comment (LJKaski) [reply tsconfig.json:50]: "--if it works" seems to be the limiting factor. Starting to look like we need to figure out another way of including the file in the build since it still doesn't work as intended when the src folder isn't present.
